packages/apps/stock/src/screens/loader/LoaderScreen.js

```diff
@@ -56,12 +56,14 @@ const LoaderScreen = () => {
     process,
     onSuccess: handleCustomAction,
     onError: () => console.warn('An error has occurred!'),
+    disabled: true,
   });
```

Comment:
The popup component should wrap all the logic, so the use of the hook should be in the popup